### PR TITLE
Added documentation to day-1 tutorial

### PR DESCRIPTION
**Pull request by mickaelandrieu**
Hi,

I've added docs related to day 1:

* Introduction to BackBee
* Prerequisites
* Installation process

In my opinion, for newbies they may need 1h or more to install BackBee the first time because they mayhave to set and configure their environment. So I think we should start to talk about application feature only on day 2.

ping @backbee/core-team 

PS: click on view button to see the real render of Markdown file.

### CODE REVIEW

**Comment by mickaelandrieu** [comment]
@fkroockmann it's captures from what you realy see, I know there is too much space in some steps maybe should we improve the installer than actualy improve the captures ? 

@ndufreche I can't make my first "contribution" as 0.11 branch is actualy broken on contribution mode /c @crouillon

**Comment by fkroockmann** [comment]
The size of picture is too small for read without click on the picture.
You can change the size of picture in markdown like this:
http://stackoverflow.com/questions/14675913/how-to-change-image-size-markdown

**Comment by mickaelandrieu** [comment]
hi @fkroockmann: I don't know if gitbhub take care of this way to do but I will try.

Then, I hope this will be merged.

Thank you all for this complete review :)

**Comment by fkroockmann** [comment]
Great job :+1:

File: doc/day1.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
+Day 1 : Project start
+================
+
+Introduction
+---------------
+
+[BackBee](http://www.backbee.com) is an open source project built on top of Symfony Components and Doctrine 2.
```
Comment (cedric-b) [inline doc/day1.md]: Eventuellement ajouter CMS => "open source CMS project" pour indiquer tout de même qu'il s'agit d'un CMS ?

Comment (mickaelandrieu) [inline doc/day1.md]: true, updated.

**Pull request merged**